src/librustdoc/html/sources.rs

```diff
@@ -113,7 +123,7 @@ impl SourceCollector<'_, 'tcx> {
         href.push_str(&fname.to_string_lossy());
 
         let title = format!("{} - source", src_fname.to_string_lossy());
-        let desc = format!("Source of the Rust file `{}`.", filename);
+        let desc = format!("Source of the Rust file `{}`.", filename.prefer_local());
```

Comment:
Are you sure prefer_local is correct? I would expect the documentation to get remapped, it's a public artifact the same way binaries are.